src/widgetastic_patternfly4/wizard.py

```diff
@@ -0,0 +1,286 @@
+    def __init__(
+        self,
+        parent,
+        start_widget,
+        footer_widgets,
+        step_view_config=None,
+        finished_view_config=None,
+        logger=None,
+    ):
+        super(Wizard, self).__init__(parent, logger=logger)
+        self._main_view = WizardMainView(
+            self.browser, step_view_config, finished_view_config, footer_widgets
+        )
+        self._start_widget = start_widget
+        self._finished_view_config = finished_view_config
+
+    @property
+    def title(self):
+        return self._main_view.header_view.title
+
+    @property
+    def subtitle(self):
+        return self._main_view.header_view.subtitle
+
+    @property
+    def steps(self):
+        return self._main_view.steps
+
+    @property
+    def is_displayed(self):
+        return self._main_view.is_displayed
+
+    @property
+    def active_step(self):
+        return self._main_view.active_step
+
+    def select_step(self, step_name):
+        self._main_view.select_step(step_name)
+
+    def _current_view(self):
+        return self._main_view.current_view
+
+    @property
+    def view(self):
+        return self._current_view()
+
+    def _click_button(self, button_type="next", wait_for_view=True):
+        buttons = {
+            "start": self._start_widget,
+            "next": self._main_view.footer_view.next_button,
+            "finish": self._main_view.footer_view.finish_button,
+            "back": self._main_view.footer_view.back_button,
+            "cancel": self._main_view.footer_view.cancel_button,
+            "close": self._main_view.header_view.close_button,
+        }
+
+        button = buttons[button_type]
+
+        if button.is_displayed and button.is_enabled:
+            button.click()
+        else:
+            raise ValueError("No button type %s found or enabled", button_type)
+
+        if wait_for_view:
+            self.view.wait_displayed(delay=1)
+
+    def start(self, wait_for_view=True):
+        self._click_button("start", wait_for_view=wait_for_view)
+
+    def finish(self, wait_for_view=False):
+        if self._finished_view_config:
+            wait_for_view = True
+        self._click_button("finish", wait_for_view=wait_for_view)
+
+    def next(self, wait_for_view=True):
+        self._click_button("next", wait_for_view=wait_for_view)
+
+    def back(self, wait_for_view=True):
+        self._click_button("back", wait_for_view=wait_for_view)
+
+    def cancel(self, wait_for_view=False):
+        self._click_button("cancel", wait_for_view=wait_for_view)
+
+    def close(self, wait_for_view=False):
+        self._click_button("close", wait_for_view=wait_for_view)
+
+
+class WizardMixin(WTMixin):
+
+    START_BUTTON = None
+    STEPS = None
+    FINISHED_VIEW = None
+    FOOTER_NEXT_BUTTON = Button("Next")
+    FOOTER_BACK_BUTTON = Button("Back")
+    FOOTER_CANCEL_BUTTON = Button("Cancel")
+    FOOTER_FINISH_BUTTON = Button("Finish")
+
+    @property
+    def wizard(self):
```

Comment:
Why was the initial starting logic of the Wizard was changed ? 

What happened to 

```
        start_wizard_widget = (
            self.START_BUTTON
            if isinstance(self.START_BUTTON, Widget)
            else Button(self.browser, self.START_BUTTON)
        )
```